osx/src/platform_osx.mm

```diff
@@ -142,4 +142,6 @@ void cancelUrlRequest(const std::string& _url) {
     }];
 }
 
+void setCurrentThreadPriority(int priority) {}
```

Comment:
There should have been a TODO - I just couldn't test it. Yes, this likely works the same for OSX.